crates/egui/src/data/input.rs

```diff
@@ -436,6 +436,8 @@ pub enum Event {
 
         /// The state of the modifier keys at the time of the event.
         modifiers: Modifiers,
+
+        text: Option<String>,
```

Comment:
Needs a docstring!

How does this compare to `Event::Text`?
When should this be used instead of `Event::Text`?
When should `Event::Text` be used instead of this?
When is it `None`?
When is it `Some("")`?

Reply:
> How does this compare to Event::Text?

It's the same, but combined. Currently if eframe detects a "text representation" (aka a key combination should actually be a character defined by the keyboard layout) eframe emits two events. Once Event::Key for the key combination, once Event::Text with the text representation of it. But this detachment makes it impossible to do custom text input handling. Because one cannot know if a key combination results in a certain "text representation" or not.